tensorboard/webapp/core/store/core_initial_state_provider.ts

```diff
@@ -28,15 +29,8 @@ export const CORE_STORE_CONFIG_TOKEN = new InjectionToken<
 export function getConfig(deepLinker: HashDeepLinker): StoreConfig<CoreState> {
   return {
     initialState: {
+      ...initialState,
```

Comment:
Is there a unit test covering getConfig()? There needs to be one that asserts `activePlugin` returned by the function reflects the return value of `deepLiner.getPluginId()` it is non-null. If there isn't a test like that, please add it.